features/features.go

```diff
@@ -468,13 +468,6 @@ var (
 						enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 						mustRegister()
 
-	FeatureGateVSphereDriverConfiguration = newFeatureGate("VSphereDriverConfiguration").
-						reportProblemsToJiraComponent("Storage / Kubernetes External Components").
-						contactPerson("rbednar").
-						productScope(ocpSpecific).
-						enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
-						mustRegister()
-
```

Comment:
For reasons beyond this PR, you can't actually remove gates at present. This must just be promoted to default and not removed. Gates will be cleaned up in a future release once they have been enabled by default for at least 2 EUS upgrades

Reply:
I see, is this documented somewhere for future myself? I was looking at this: https://github.com/openshift/enhancements/blob/b99a80d976385faa87f251dbbcd4260a37406921/dev-guide/featuresets.md#steps-to-remove-a-feature-gate

Which says the feature gate can be removed once it's no longer used, and this one in particular is not used currently. Also mentions removing if clauses for feature gates `In the next release` -> so it's actually 2 EUS upgrades as you said. Is the doc outdated perhaps?